src/lib/db.ts

```diff
@@ -30,10 +30,38 @@
 import Knex from 'knex'
 import Config from '../../config/knexfile'
 import ConsentDB from '../model/consent'
-
+import {ScopeDB, Scope} from '../model/scope'
 const Db: Knex = Knex(Config.test)
 const consentDB: ConsentDB = new ConsentDB(Db)
 
+/**
+ * Retrieves scopes from database, reformats object structure, removing
+ * scope & consent ids, and returns array of scopes
+ * @param id Consent id
+ */
+export async function retrieveScopes (id: string): Promise<Scope[]> {
```

Comment:
Do we need this function here? Seems like domain logic for this specific handler. Also, this file was just to make the DB connection.

Reply:
It is domain logic, let move it there. there is a need to rethink the usage and return type, where this